skyvern/forge/sdk/routes/workflow_copilot.py

```
@@ -0,0 +1,348 @@
+        raise
+    except LLMProviderError as e:
+        LOG.error(
+            "LLM provider error",
+            organization_id=organization.organization_id,
+            error=str(e),
+            exc_info=True,
+        )
+        raise HTTPException(
+            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            detail="Failed to process your request. Please try again.",
+        )
```

Ruff (0.14.10)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)